packages/@aws-cdk/aws-codedeploy/lib/lambda/deployment-group.ts

```diff
@@ -0,0 +1,196 @@
+import cloudwatch = require('@aws-cdk/aws-cloudwatch');
+import iam = require('@aws-cdk/aws-iam');
+import lambda = require('@aws-cdk/aws-lambda');
+import cdk = require('@aws-cdk/cdk');
+
+import { CfnDeploymentGroup } from '../codedeploy.generated';
+import { AutoRollbackConfig, DeploymentOption, DeploymentType } from '../config';
+import { deploymentGroupNameToArn, renderAlarmConfiguration, renderAutoRollbackConfiguration } from '../utils';
+import { ILambdaApplication, LambdaApplication } from './application';
+import { LambdaDeploymentConfig } from './deployment-config';
+
+export interface ILambdaDeploymentGroup extends cdk.IConstruct {
+  readonly application: ILambdaApplication;
+  readonly deploymentGroupName: string;
+  readonly deploymentGroupArn: string;
+
+  export(): LambdaDeploymentGroupImportProps;
+}
+
+export interface LambdaDeploymentGroupProps {
+  application: LambdaApplication;
+  deploymentGroupName?: string;
+  deploymentConfig: LambdaDeploymentConfig;
```

Comment:
Again, this should be optional. Check with the docs what is the default - if it's required, we should pick a sensible default.